modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -281,315 +228,276 @@ This role that the EFS CSI controller uses can now assume a role inside account
         }
     ]
 }
+EOF
 ----
 
-.. Create the policy.
+.. Create the IAM policy by running the following command:
 +
 [source,terminal]
 ----
-$ aws iam create-policy --policy-name "cross-account-rosa-efs-csi" \
-   --policy-document file://<path-to-file>.json \
-   --query 'Policy.Arn' --output text
+ACCOUNT_B_POLICY_ARN=$(aws iam create-policy --policy-name "${CLUSTER_NAME}-efs-csi-policy" \
+   --policy-document file://$SCRATCH_DIR/EfsPolicyInAccountB.json \
+   --query 'Policy.Arn' --output text) \
+&& echo ${ACCOUNT_B_POLICY_ARN}
 ----
 
-.. Create a trust policy.
+.. Attach the policy to the role by running the following command:
 +
-[source, json]
+[source,terminal]
 ----
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Principal": {
-                "AWS": "arn:aws:iam::${AWS_ACCOUNT_A_ID}:root"
-            },
-            "Action": "sts:AssumeRole",
-            "Condition": {}
-        }
-    ]
-}
+aws iam attach-role-policy \
+   --role-name "${ACCOUNT_B_ROLE_NAME}" \
+   --policy-arn "${ACCOUNT_B_POLICY_ARN}"
+----
+
+. Configure the AWS Account A IAM roles and policies:
+
+.. Switch to your Account A profile by running the following command:
++
+[source,terminal]
+----
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_A}
 ----
 
-.. Create the role for the EFS CSI Driver Operator to assume.
+.. Create the IAM policy document by running the following command:
 +
 [source,terminal]
 ----
-$ aws iam create-role \
-  --role-name "cross-account-efs-role" \
-  --assume-role-policy-document file://<path-to-file>.json \
-  --query "Role.Arn" --output text
+cat << EOF > $SCRATCH_DIR/AssumeRoleInlinePolicyPolicyInAccountA.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "sts:AssumeRole",
+      "Resource": "${ACCOUNT_B_ROLE_ARN}"
+    }
+  ]
+}
+EOF
 ----
 
-.. Attach the policies to the role.
+.. In AWS Account A, attach the AWS-managed policy "AmazonElasticFileSystemClientFullAccess" to the {product-title} cluster master role by running the following command:
 +
-[source, terminal]
+[source,terminal]
 ----
-$ aws iam attach-role-policy \
---role-name "cross-account-efs-role" \
---policy-arn ${ACCOUNT_B_POLICY_ARN}
+EFS_CLIENT_FULL_ACCESS_BUILDIN_POLICY_ARN=arn:aws:iam::aws:policy/AmazonElasticFileSystemClientFullAccess
```

Comment:
```suggestion
EFS_CLIENT_FULL_ACCESS_BUILTIN_POLICY_ARN=arn:aws:iam::aws:policy/AmazonElasticFileSystemClientFullAccess
```
typo as @RomanBednar mentioned in https://github.com/openshift/openshift-docs/pull/97085#discussion_r2262662478